src/languages/php.md

```diff
@@ -140,6 +140,46 @@ This is the complete list of extensions that can be enabled:
 > see the up-to-date complete list of extensions after you SSH into
 > your environment. For PHP 7, use `ls /etc/php/*/mods-available`.
 
+## Custom PHP extensions
+
+It is possible to use an extension not listed here, but there are a few steps involved.
+
+1. Either download the extension .so file in your build hook or Git commit that to your Git repository.
```

Comment:
Actually, committing a blob into Git is against the industry practice, let's not suggest it here :p Just ask user to download the extension in build hook :)